test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/SystemDataSqlClientTests.cs

```diff
@@ -60,5 +63,28 @@ public void SubmitsTracesWithNetStandard(string packageVersion)
                 }
             }
         }
+
+        [Fact]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        public void SpansDisabledByAdoNetExcludedTypes()
```

Comment:
Should we add a similar test to `MicrosoftDataSqlClientTests` or does this cover both?